iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/user/detail/user.detail.component.html

```diff
@@ -83,12 +83,18 @@ <h3 class="profile-username text-center">{{$ctrl.user.name.formatted}}</h3>
                 </td>
             </tr>
             <tr ng-if="$ctrl.aupIsEnabled()">
-                <td class="text-bold text-left">
+                <td class="text-bold text-left" ng-if="!$ctrl.user.aupSignatureExpiryDate">
                     Signed AUP
                 </td>
-                <td class="text-right" ng-if="$ctrl.hasAupSignatureTime()">
+                <td class="text-bold text-left" ng-if="$ctrl.user.aupSignatureExpiryDate">
+                    AUP signature expire in
```

Comment:
typo: expires

Reply:
I wanted to say: `AUP signature will expire in`, shall I add `will`?

Reply:
i's a bit long :) We can leave it as it is or at least a shorter "AUP expiration"? what do you think?